### PR TITLE
update jest junit configuration

### LINKED ISSUE #452: Update jest junit to be compatible and consistent with the main generator
Just make sure we use the same junit-jest reporter version and configuration to not interfere with e.g. ci script from the main generator.

https://github.com/jhipster/generator-jhipster/pull/10473

### PR DESCRIPTION
**Pull request by atomfrede**
Align with main generator configuration

closes #452

-   Please make sure the below checklist is followed for Pull Requests.

-   [ ] [Azure tests](https://dev.azure.com/jhipster/jhipster-vuejs/_build) are green
-   [ ] Tests are added where necessary
-   [ ] Documentation is added/updated where necessary
-   [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/jhipster-vuejs/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

File: generators/client/templates/vue/package.json.ejs [hunk 1]
```diff
@@ -129,7 +129,8 @@ limitations under the License.
     "typescript": "3.2.4",
     "uglifyjs-webpack-plugin": "2.0.1",
     "url-loader": "1.1.2",
-    "vue-jest": "3.0.3",
+    "babel-core": "^7.0.0-bridge.0",
```
Comment (pascalgrimaud) [inline generators/client/templates/vue/package.json.ejs]: - do we need babel ?
- can you remove ^ to use a fixed version ?
- maybe order it alphabetically

Comment (atomfrede) [reply generators/client/templates/vue/package.json.ejs]: According to the [documentation](https://github.com/vuejs/vue-jest/tree/master#usage-with-babel-7) I also thought we don't need it, but it seems there is a little problem with peer [dependencies](https://github.com/vuejs/vue-jest/issues/160).

Sure the `^` is not required. Will also sort it alphabetically (put it there as it "belongs" to vue-jest).

Comment (vishal423) [reply generators/client/templates/vue/package.json.ejs]: not sure if I understood correctly, but, it seems you don't have to as you are upgrading to use jest > `v24` and also don't see any `babel-jest` dependency here:
> If you use jest < 24.0.0 and babel-jest make sure to install babel-core@bridge

Comment (vishal423) [reply generators/client/templates/vue/package.json.ejs]: ignore my last comment :smile:

Comment (atomfrede) [reply generators/client/templates/vue/package.json.ejs]: I guess you are likewise confused like I was when reading the sentence and looking at our issue :man_shrugging:

**Pull request merged**